pandas/tests/indexing/multiindex/test_slice.py

```diff
@@ -744,3 +744,11 @@ def test_non_reducing_slice_on_multiindex(self):
         result = df.loc[tslice_]
         expected = pd.DataFrame({("b", "d"): [4, 1]})
         tm.assert_frame_equal(result, expected)
+
+    def test_truncate_multiindex_names(self):
```

Comment:
Instead of here you'll likely want to add tests in /tests/{series,frame}/methods/test_truncate.py and then test the whole Series / DataFrame output, which will also test the attributes (e.g., names) of the indexes.

Reply:
yeah just grep for the truncate tests and co-locate this there (and need one for frame and similar test for series)